playground/infrastructure/datastore_client.py

```diff
@@ -360,7 +360,7 @@ def _to_dataset_snippet_entity(self, dataset_id: str, snippet_id: str, emulator:
                 "dataset": self._get_dataset_key(dataset_id),
                 "snippet": self._get_snippet_key(snippet_id),
                 "emulator": emulator.name,
-                "config": emulator_config
+                "config": str(emulator_config).replace("'", "\"")
```

Comment:
looks cryptic, why we need this?

Reply:
It needs to process this field as json in golang:
```
	var configInterface map[string]interface{}
	if err := json.Unmarshal([]byte(datasetSnippet.Config), &configInterface); err != nil {
		return nil, err
	}
```
I added this replacing here because we make this one time before saving to the cloud datastore